coursemapper-kg/concept-map/src/pipelines/concept_map.py

```
@@ -166,13 +166,13 @@
 
             for keyphrase, annotation, page_title, page_categories, page_summary, page_embedding, weight, material_weight in disambiguated_df.iter_rows():
                 # Add concept node to graph
-                concept_node = Node(f'{material_id}_concept_{str(abs(hash(str(page_embedding))))}', page_title, '', 'main_concept', (material_weight + 1) / 2, f'https://en.wikipedia.org/wiki/{page_title}', page_summary, False, page_embedding)
+                concept_node = Node(f'{material_id}_concept_{str(abs(hash(str(page_embedding))))}', page_title, '', 'main_concept', (material_weight + 1) / 2, f'https://en.wikipedia.org/wiki/{page_title}', page_summary, False,False,False, page_embedding)
```

Ruff (0.8.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)